### PR TITLE
Migration and Model for Product and Category

### PR DESCRIPTION
**Pull request by wowKimsohyun17**
Create database and model for product and category

### CODE REVIEW

**Comment by wowKimsohyun17** [comment]
ready

File: app/Product.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Product extends Model
+{
+    //
+    public function productCategory()
```
Comment (huyenttm-0901) [inline app/Product.php]: bỏ comment

File: app/Product.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App;
```
Comment (huyenttm-0901) [inline app/Product.php]: đưa model vào folder Models

File: app/ProductCategory.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class ProductCategory extends Model
+{
+    //
+    public function products()
```
Comment (huyenttm-0901) [inline app/ProductCategory.php]: bỏ comment

File: app/ProductCategory.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class ProductCategory extends Model
```
Comment (huyenttm-0901) [inline app/ProductCategory.php]: bảng trung gian ko có model nhé

**Comment by wowKimsohyun17** [comment]
ready

File: app/Models/Category.php [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
+{
```
Comment (huyenttm-0901) [inline app/Models/Category.php]: thêm fillable đi e

**Comment by wowKimsohyun17** [comment]
ready

File: app/Models/Category.php [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+class Category extends Model
+{   
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'name'
```
Comment (thaonp104) [inline app/Models/Category.php]:
```suggestion
        'name',
```

File: app/Models/Product.php [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'name', 
+        'price',
+        'description',
+        'img_url',
+        'category_id'
```
Comment (thaonp104) [inline app/Models/Product.php]:
```suggestion
        'category_id',
```

**Review by huyenttm-0901 — COMMENTED** [review]
(merged)

**Pull request merged**